### PR TITLE
Add bleeping

### PR DESCRIPTION
**Pull request by jsvana**
`<prefix>bleep <bad_word> <replacement>` will add a new bleep, then
saying `<bad_word>` will get a reply that you should use `<replacement>`
instead.

### CODE REVIEW

File: seabird/modules/bleep.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+class BleepPlugin(Plugin, CommandMixin, DatabaseMixin):
+    REPLIES = [
+        'Hey, watch your mouth! Say "{}" instead.',
+        'Pottymouth! We say "{}" in this channel.',
+        'Uh oh, you should really say "{}" instead.',
+        'Time to put a quarter in the jar! You should really use "{}" instead.',
+    ]
+
+    def _get_bleeps(self):
```
Comment (belak) [inline seabird/modules/bleep.py:28]: I'd like to see this cached in the class

**Comment by belak** [comment]
Should bleeps be global or on a per-channel basis?

**Comment by jsvana** [comment]
Global. If people really want this per-channel they can file an issue.

File: seabird/modules/bleep.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+            )
+            return
+
+        bad_word = args[0]
+        replacement = args[1]
+
+        bleep = Bleep()
+        bleep.bad_word = args[0]
+        bleep.replacement = args[1]
+        with self.db.session() as session:
```
Comment (belak) [inline seabird/modules/bleep.py:64]: I specifically implemented this in the DB session so we wouldn't have to keep doing this. There's get_or_create on the session object. I believe it's used in the karma module.

**Comment by jsvana** [comment]
Made it use get_or_create.

**Pull request merged**